NanoAnalysis/python/ZZFiller.py

```diff
@@ -351,28 +354,34 @@ def analyze(self, event):
 
         ### Now fill the variables to be stored as output
         # Fill selected Zs
-        ZCand_mass = []
-        ZCand_flav = []
-        ZCand_l1Idx = []
-        ZCand_l2Idx = []
-        ZCand_fsr1Idx = []
-        ZCand_fsr2Idx = []
+        ZCand_mass = [0.]*len(SRZs)
+        ZCand_pt = [0.]*len(SRZs)
+        ZCand_eta = [0.]*len(SRZs)
+        ZCand_rapidity = [0.]*len(SRZs)
+        ZCand_phi = [0.]*len(SRZs)
+        ZCand_flav = [0.]*len(SRZs)
+        ZCand_l1Idx = [0.]*len(SRZs)
+        ZCand_l2Idx = [0.]*len(SRZs)
```

Comment:
Idx variables are integers so they should be initialized with [0] and not with [0.] (it should spare a conversion...)